### PR TITLE
Shl Update somatic short variants tool docs

### PR DESCRIPTION
**Pull request by sooheelee**
Tools touched:

 Mutect2
 CreateSomaticPanelOfNormals
 GetPileupSummaries
 CalculateContamination
 FilterMutectCalls
 Picard CollectSequencingArtifactMetrics
 FilterByOrientationBias
 Concordance 

The Mutect2 (How to) tutorial and various supporting documents are now public and so I've cleaned up now redundant information we temporarily placed in the tool docs. I also made a pass at unifying this set of tool docs in terms of presentation elements.

### CODE REVIEW

**Comment by sooheelee** [comment]
@davidbenjamin, @takutosato @vdauwera, would one of you be willing to review these minor updates? In particular, FilterByOrientationBias has a three-bullet list at the end that I summarized out of the detailed summary section. It seems the detailed summary section points may be out of date (a feeling of deja vu here also). Let me take a screenshot so you can see:

screenshot 2018-01-30 17 48 24

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2.java [hunk 1]
```diff
@@ -18,21 +18,51 @@
...
+ *     See <a href="https://software.broadinstitute.org/gatk/documentation/article?id=11136">Tutorial#11136</a> for a
+ *     step-by-step description of the workflow and <a href="https://software.broadinstitute.org/gatk/documentation/article?id=11127">Article#11127</a>
+ *     for an overview of what traditional somatic calling entails. For the latest pipeline scripts, see the
+ *     <a href="https://github.com/broadinstitute/gatk/tree/master/scripts/mutect2_wdl">Mutect2 WDL scripts directory</a>.
+ *     Although we present the tool for somatic calling, it may apply to other contexts, such as mitochondrial variant calling.
  * </p>
  *
- * <p>For how GATK4 Mutect2 differs from GATK3 MuTect2, see <a href='https://software.broadinstitute.org/gatk/documentation/article?id=10911'>GATK Article#10911</a>.</p>
+ * <h3>Usage examples</h3>
+ * <p>Example commands show how to run Mutect2 for typical scenerios. The two modes are (i) <i>somatic mode</i> where a tumor sample is matched with a normal sample in analysis and
```
Comment (davidbenjamin) [inline src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2.java]: Typo: scenerios -> scenarios

**Comment by sooheelee** [comment]
Thanks for the review @davidbenjamin. I just read the release notes for v4.0.1.0 and saw that there is a fix to allow `.list` extensions in addition to `.arg` extensions, so I added this to CreateSomaticPanelOfNormals.

**Comment by sooheelee** [comment]
Given the thumbs up from David, I will go ahead and merge these doc updates.

**Pull request merged**